articles/partner-solutions/dynatrace/index.yml

```
@@ -15,11 +15,11 @@ metadata:
 
 landingContent:
   # Card (optional)
-  - title: About Dynatrace for Azure 
+  - title: About Azure Native Dynatrace Service
```

```suggestion
  - title: About the Azure Native Dynatrace Service
```
preference, not required